docs/components/Banner/Banner.stories.mdx

```diff
@@ -18,8 +18,8 @@ Banner should align to the left and right of the content it is related to.
 
 When using a Banner within a modal or a page layout, this means Banner should
 align with the text, inputs or other UI elements beneath it. For best results,
-use [Content](../?path=/docs/components-layouts-and-structure-content--docs) to
-ensure consistent vertical spacing between Banner and adjacent elements.
+use [Content](/components/Content) to ensure consistent vertical spacing between
+Banner and adjacent elements.
```

Comment:
Further down, from lines 41-44 there is a list of banner types:

- `success` (web only) [<Icon name="link" />](#success-message)
- `notice` [<Icon name="link" />](#notice-message)
- `warning`[<Icon name="link" />](#warning-message)
- `error` [<Icon name="link" />](#error-message)

Those have links to sections within the Storybook page. I suggest we take out the list with the Icon and link and just list the types. So on line 39 we put something like:

`Banners come in four types; success, notice, warning and error. Each type has guidelines for usage:`
and then we go straight to the next heading
`### Success message (Web only)`